flink-runtime/src/main/java/org/apache/flink/runtime/metrics/groups/JobManagerMetricGroup.java

```diff
@@ -95,6 +96,15 @@ public int numRegisteredJobMetricGroups() {
 		return jobs.size();
 	}
 
+	// ------------------------------------------------------------------------
+	//  Component Metric Group Specifics
+	// ------------------------------------------------------------------------
+
+	@Override
+	protected void putVariables(Map<String, String> variables) {
+		variables.put(ScopeFormat.SCOPE_ACTOR_HOST, hostname);
```

Comment:
Can we call this simply `SCOPE_HOST`? The "actor" part is an implementation specific detail that should not be part of this abstraction level.